ui/apps/platform/src/Containers/VulnMgmt/Dashboard/VulnMgmtDashboardPage.js

```diff
@@ -72,7 +76,15 @@ const VulnDashboardPage = ({ history }) => {
             <div className="flex items-center">
                 <div className="flex h-full mr-3 pr-3 border-r-2 border-base-400">
                     <PoliciesCountTile />
-                    <CvesCountTile />
+                    {showVmUpdates ? (
+                        <>
+                            <CvesCountTile entityType={entityTypes.IMAGE_CVE} />
+                            <CvesCountTile entityType={entityTypes.NODE_CVE} />
+                            <CvesCountTile entityType={entityTypes.CLUSTER_CVE} />
+                        </>
```

Comment:
The only issue with having multiple tiles is that it'll push the export button offscreen and look a bit odd

<img width="1552" alt="Screen Shot 2022-06-30 at 12 15 54 PM" src="https://user-images.githubusercontent.com/4805485/176759764-2967555e-c08e-4af7-ac38-525838b55916.png">

cc: @alanonthegit

Reply:
@alanonthegit I had an idea for handling the links to the three new types of CVE lists, without having to re-engineer the entire top bar.

How about I make the CVE links a dropdown list of links, like the *Applications & Infrastructure" list of links?